hathor/transaction/scripts/execute.py

```diff
@@ -88,7 +107,7 @@ def evaluate_final_stack(stack: Stack, log: list[str]) -> None:
         raise FinalStackInvalid('\n'.join(log))
 
 
-def script_eval(tx: Transaction, txin: TxInput, spent_tx: BaseTransaction) -> None:
+def script_eval(tx: Transaction, spent_tx: BaseTransaction, *, input_index: int) -> 'ScriptContext':
```

Comment:
As you're already changing the method signature, you can put it all as kwargs only, right?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/911/commits/2f8579f7c83882245a077f7d354f97457d51925e